pkg/uploader/uploader.go

```diff
@@ -51,7 +51,7 @@ func NewUploaderWithClient(log *libbuildpack.Logger, client AMSClient) Uploader
 	}
 }
 
-func (up *uploader) Upload(rootDir string, dstURL string) error {
+func (up *uploader) Upload(rootDir string, dstURL string, amsInstanceID string) error {
```

Comment:
I would rename amsInstanceId to authzInstanceId

Reply:
The naming (ams vs. authorization) is not really consistent :/ 
I left it at `amsInstanceID`, since the AMS endpoint is also uses `.../ams-instances/<ID>`.